packages/network/src/logic/Tracker.ts

```diff
@@ -254,4 +254,12 @@ export class Tracker extends EventEmitter {
     getOverlayPerStream(): Readonly<OverlayPerStream> {
         return this.overlayPerStream
     }
+
+    getTrackerRecord(): TrackerRecord {
+        return {
+            id: this.peerInfo.peerId,
+            http: this.getUrl().replace(/^ws/, 'http'),
```

Comment:
If the tracker doesn't have HTTP endpoints, this field should be `undefined`? See https://github.com/streamr-dev/network-monorepo/blob/06137024a0e56bd24e601fe27ac6811d0ffe3dc6/packages/network/src/composition.ts#L90

Currently the `TrackerRecord` type definition doesn't allow `undefined` field value. Does the smart contract allow it?

Reply:
> if the tracker doesn't have HTTP endpoints

Good point

Reply:
> Does the smart contract allow it?

Supports anything, the data is a json blob as a string.